backend/app/cmd/server.go

```diff
@@ -179,14 +179,16 @@ type NotifyGroup struct {
 		API     string        `long:"api" env:"API" default:"https://api.telegram.org/bot" description:"telegram api prefix"`
 	} `group:"telegram" namespace:"telegram" env-namespace:"TELEGRAM"`
 	Email struct {
-		Host                string        `long:"host" env:"HOST" description:"SMTP host"`
-		Port                int           `long:"port" env:"PORT" default:"587" description:"SMTP port"`
-		TLS                 bool          `long:"tls" env:"TLS" description:"enable TLS for SMTP"`
-		From                string        `long:"fromAddress" env:"FROM" description:"from email address"`
-		Username            string        `long:"username" env:"USERNAME" description:"SMTP user name"`
-		Password            string        `long:"password" env:"PASSWORD" description:"SMTP password"`
-		TimeOut             time.Duration `long:"timeout" env:"TIMEOUT" default:"10s" description:"SMTP TCP connection timeout"`
-		VerificationSubject string        `long:"verification_subj" env:"VERIFICATION_SUBJ" description:"verification message subject"`
+		Host                     string        `long:"host" env:"HOST" description:"SMTP host"`
+		Port                     int           `long:"port" env:"PORT" default:"587" description:"SMTP port"`
+		TLS                      bool          `long:"tls" env:"TLS" description:"enable TLS for SMTP"`
+		From                     string        `long:"fromAddress" env:"FROM" description:"from email address"`
+		Username                 string        `long:"username" env:"USERNAME" description:"SMTP user name"`
+		Password                 string        `long:"password" env:"PASSWORD" description:"SMTP password"`
+		TimeOut                  time.Duration `long:"timeout" env:"TIMEOUT" default:"10s" description:"SMTP TCP connection timeout"`
+		VerificationSubject      string        `long:"verification_subj" env:"VERIFICATION_SUBJ" description:"verification message subject"`
+		VerificationTemplatePath string        `long:"verification_tmpl_path" env:"VERIFICATION_TMPL_PATH" description:"path to verification email template"`
```

Comment:
Missing default

Reply:
And I need to remove it from defaultEmailTemplatePath in email.go?

Reply:
No, it's additional safeguard.

Reply:
I did like that just because VerificationSubject also isn't have default

Reply:
The subject is complex and constructed out of site name if I'm not mistaken, that's the only reason it's not written to default.

Reply:
Btw it's written to Readme, maybe incorrectly, but written.